docs/pages/experiments/material-ui/css-variables.tsx

```diff
@@ -41,56 +42,162 @@ export default function Page() {
         <Box sx={{ pb: 4 }}>
           <ColorSchemePicker />
         </Box>
-        <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 4, mb: 1 }}>
-          <Button variant="contained">Text</Button>
-          <Button variant="outlined">Text</Button>
-          <Button>Text</Button>
-        </Box>
-        <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 4, mb: 1 }}>
-          <Button color="secondary" variant="contained">
-            Text
-          </Button>
-          <Button color="secondary" variant="outlined">
-            Text
-          </Button>
-          <Button color="secondary">Text</Button>
-        </Box>
-        <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 4, mb: 1 }}>
-          <Button color="error" variant="contained">
-            Text
-          </Button>
-          <Button color="error" variant="outlined">
-            Text
-          </Button>
-          <Button color="error">Text</Button>
-        </Box>
-        <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 4, mb: 1 }}>
-          <Button color="info" variant="contained">
-            Text
-          </Button>
-          <Button color="info" variant="outlined">
-            Text
-          </Button>
-          <Button color="info">Text</Button>
-        </Box>
-        <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 4, mb: 1 }}>
-          <Button color="warning" variant="contained">
-            Text
-          </Button>
-          <Button color="warning" variant="outlined">
-            Text
-          </Button>
-          <Button color="warning">Text</Button>
-        </Box>
-        <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 4, mb: 1 }}>
-          <Button color="success" variant="contained">
-            Text
-          </Button>
-          <Button color="success" variant="outlined">
-            Text
-          </Button>
-          <Button color="success">Text</Button>
-        </Box>
+        <div>
```

Comment:
It was wrapped in a `div` to be able to better divide the content and to be able to collapse by section.

Reply:
This was reversed